Added location starting location description (#139)
ezsystems/user-documentation: docs/site_organization/organizing_the_site.md

    You can delete a Content Type only when there are no Content items that belong to it.
    This also includes Content items in the Trash.

### Setting relation starting location

[Relation Field types](../creating_content_basic.md) allow to configure the location in the Content tree which opens in the Universal Discovery Widget (UDW).

> To set the starting location for the relation, perform the following steps. 

Author: I will stick to "steps".

> Go to **Content Types** and edit the respective Content Type.
> Next, go to **Field definitions**. In the `objectrelation` Field, select from the available options:

Reviewer: ```suggestion

1. Go to **Content Types** and edit the respective Content Type.
2. Go to **Field definitions** and either select or add the `objectrelation` Field
3. Under **Select starting location**, select from the available options:
```

Author: I follow the pattern: one sentence, one procedure. Using 'or' is not the best word as UX is concerned.

Reviewer: The point is that the `objectrelation` field could be missing (like when you create a new content type or develop an existing one) and the admin might want to add it. This is not UX, this is how it may look in the product and in my opinion we should account for differing conditions.

> - **Default** - the starting location is automatically assigned to the default location in the tree of a created Content.

Reviewer: ```suggestion
- **Default** - the starting location is automatically assigned to the default location in the tree of a created Content.
```

Content item or Content type?

> - **Default** - the starting location is automatically assigned to the default location in the tree of a created Content Type.

Reviewer 2: ```suggestion
- **Default** - the starting location is automatically assigned to the default location in the tree of a created Content.
```

> - **Browse** - use to manually select the location from the Universal Discovery Widget.

Reviewer: ```suggestion
- **Browse** - use to manually select the location from the Content Browser.
```

> - **Content location** - the starting location is the location of the currently edited Content. For example, if you edit the Content item with the location `50`, it sets the starting location to this value with children under this location.

Reviewer: ```suggestion
- **Content location** - the starting location is the location of the Content item that is edited by the user. For example, if the user edits the Content item with the location `50`, it sets the starting location to this value with children under this location.
```

"currently edited Content" - item or type? Is the content type currently edited by the administrator user, or a content item currently edited by the end user?

> - **Content location** - the starting location is the location of the currently edited Content. For example, if you edit the Content Type with the location `50`, it sets the starting location to this value with children under this location.

Reviewer 2: ```suggestion
- **Content location** - the starting location is the location of the currently edited Content. For example, if you edit the Content item with the location `50`, it sets the starting location to this value with children under this location.
```

> - **Root default location** - use if you want the Universal Discovery Widget to start with the defined location with only children available for selection.

Reviewer: ```suggestion
- **Root default location** - use if you want the Content Browser to start at the defined location with only children available for selection.
```
